stdlib/public/Concurrency/CheckedContinuation.swift

```diff
@@ -296,4 +299,65 @@ public func withCheckedThrowingContinuation<T>(
     body(CheckedContinuation(continuation: $0, function: function))
   }
 }
+#else
```

Comment:
Have you considered using [SE-0367](https://github.com/apple/swift-evolution/blob/main/proposals/0367-conditional-attributes.md)? I _think_ using that you can conditionally add the `available` attribute on the main declaration and avoid having to duplicate it.

So for example just above the line `public struct CheckedContinuation`, you can add:

```swift
#if SWIFT_STDLIB_TASK_TO_THREAD_MODEL_CONCURRENCY
  @available(*, unavailable, message: "Unavailable in task-to-thread concurrency model")
#endif
```

Reply:
The implementation also changes.